source/templates/code_signing.slim

```diff
@@ -106,24 +106,30 @@
 						.menu.small
 							select.menu-selector[ng-model="provProfileCtrl.selectedMenu" ng-model-options="{getterSetter: true}" ng-options="menu.title for menu in provProfileCtrl.menus" ng-class="{ 'selected' : menu == provProfileCtrl.selectedMenu }"]
 					.content.vertical-scroll[ng-switch="provProfileCtrl.selectedMenu.id"]
+						.error-notice[ng-if="!provProfileCtrl.hasValidIdentity()" ng-switch-when="matching-certificates"]
+							div == data[:strings][:code_signing][:prov_profile][:prov_profile_details_popup][:matching_certificates_details][:missing_or_expired_notice_part_1]
```

Comment:
Two sentences - should either be `span` and `strong`, or plain text and `strong` or two `p`s, one of them having a `strong` inside, holding the text.